core/src/main/java/io/aiven/klaw/dao/migration/MigrateData2x4x0.java

```diff
@@ -86,4 +88,44 @@ private void migrateKwProperties(Integer tenantId) {
       log.error("Exception caught: ", ex);
     }
   }
+
```

Comment:
line 79 properties.remove(propertyToUpdate); seems to have a problem. propertyToUpdate is not of type KwProperties